### PR TITLE
Assert checkData

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Review by sachseka — APPROVED** [review]
LGTM!

File: R/checkData.R
Comment (sachseka) [inline R/checkData.R:1]: LGTM! but please pay attention to tests in corresponding testthat-file. if not all checks will fail

**Review by sachseka — CHANGES_REQUESTED** [review]

**Comment by sachseka** [comment]
Please pay attention to checks

**Comment by grewered** [comment]
error: lists/data frames are type character - see https://github.com/sachseka/eatPrep/pull/25

**Review by sachseka — APPROVED** [review]

**Pull request merged**